src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+interface Props {
+  people: Person[]
+  selectedSlug: string;
+}
+
+export const PeopleTable: React.FC<Props> = ({ people, selectedSlug }) => {
+  const findParent = (
+    ArrayOfPeople: Person[],
+    parent: string | null,
+  ):Person | null => {
+    return ArrayOfPeople.find(user => user.name === parent) || null;
+  };
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {people.map(person => {
+          const {
+            sex,
+            born,
+            died,
+            motherName,
+            fatherName,
+            slug,
+          } = person;
+
+          const mother = findParent(people, motherName);
+          const father = findParent(people, fatherName);
```

Comment:
Use this functions to prepare array from server, now it will find objects on every render, let's avoid it